pom.xml

```diff
@@ -32,8 +32,8 @@
 
     <xbean.version>4.20</xbean.version>
     <owb.version>2.0.23</owb.version>
-    <tomcat.version>10.0.6</tomcat.version>
-    <johnzon.version>1.2.13</johnzon.version>
+    <tomcat.version>10.1.0-M2</tomcat.version>
```

Comment:
we should stick to 10.0 branch since 10.1 is about a spec not yet finished IMHO

Reply:
@rmannibucau downgrade to 10.0.8 done